Microsoft.Toolkit.Uwp.UI.Controls/AdaptiveGridView/AdaptiveGridView.cs

```diff
@@ -58,6 +65,13 @@ private void RecalculateLayout(double containerWidth)
             }
 
             ItemWidth = (containerWidth / _columns) - 5;
+
+            if (ItemAspectRatio > 0)
+            {
+                var safeRatio = Math.Min(100.0, Math.Max(0.01, ItemAspectRatio));
+                _templateProxy.Height = ItemWidth / safeRatio;
+                ItemHeight = _templateProxy.Height;
```

Comment:
Hmm, true.  I don't have enough experience with DependencyProperty-based UI frameworks to know the right approach here.  The problem comes about because an item's height is related to its aspect ratio - changing one would naturally change the other.

Should I break the cycle by introducing a distinction between desired/actual, like FrameworkElement.ActualHeight does?

Or should I give up on trying to report the resulting ItemHeight back to the app in any way, and just allow Aspect and Height get into an inconsistent state?